### DISCUSSION TITLE
Bug: Ability to change entire NavbarElement like v7

### DISCUSSION THREAD

**Discussion by lloydstubber** [gpbl/react-day-picker#1450] [Support]
### Bug description

First of all, thanks for the great work on the library!

I'm bumping v7 to v8 and we're using a custom Navbar, as we want an icon on either side of the caption. Are there plans for the CustomComponents to accept an entire Navbar like the previous version? Or potentially more fine grain ability to control the order of icons/caption?

### Screenshots

Here is our currently implementation in v7 and struggling to obtain it with v8.
<img width="402" alt="Screen Shot 2022-04-06 at 11 05 24 am" src="https://user-images.githubusercontent.com/18339250/161875416-b47f66e5-ec42-4519-90bc-8d6379621cfe.png">

**Comment by Michael-Nielsen** [comment]
I've created a quick demo of how to do this here: https://codesandbox.io/s/nice-snowflake-l6urs8?file=/src/App.tsx

The Codesandbox is largely based on what is already being done in files such as https://github.com/gpbl/react-day-picker/blob/master/packages/react-day-picker/src/components/Navigation/Navigation.tsx

**Comment by gpbl** [comment]
Thanks @lloydstubber for your feedback. I think a `captionLayout="centered"` or similar could help with this. @Michael-Nielsen is a possible solution but I'd prefer have a cleaner API.

**Comment by caydgreen** [comment]
I was able to achieve the split arrow navbar by overriding the css.  You can see my demo here: https://codesandbox.io/s/sandpack-project-forked-u18ty1?file=/styles.css
